types/yeoman-assert/index.d.ts

```diff
@@ -0,0 +1,188 @@
+// Type definitions for yeoman-assert 3.1
+// Project: https://github.com/yeoman/yeoman-assert
+// Definitions by: Toilal <https://github.com/Toilal>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+// copy/paste of node 'assert' module definitions as export * doesn't work with module.
```

Comment:
It seems like you're trying to extend the `assert` module.
In that case, you should follow [this template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-plugin-d-ts.html).

Reply:
My mistake; that's not quite what's going on here. I will follow up with a better suggestion.

Reply:
Alright, I think this will work better:

```ts
import assert = require("assert");

declare module "assert" {
    export function file(path: string | string[]): void;
}

export = assert;
```